Automattic/wp-calypso
# JP Disconnect Survey: Add Free-form Text Input for 'Another Reason'

Add a dedicated step for the 'Another Reason' option. Background: p7rd6c-19A-p2

Question: Do we need to sanitize/normalize user input?

![image](https://user-images.githubusercontent.com/96308/33325727-64af6fb6-d453-11e7-945c-c069d0822e52.png)

To test:
* Start at `/settings/disconnect-site/:site`, go with the flow.
* On the confirmation page, before confirming, open your dev console, switch to the 'Network' tab, and clear it.
* Cmd+F for the `survey` endpoint in the console. Have a look at the payload at the very bottom: it should contain the reason.

![image](https://user-images.githubusercontent.com/96308/33325956-f4f54794-d453-11e7-831e-06f2a1cf87b0.png)
